tests/migration_test.go

```diff
@@ -1575,6 +1575,21 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 					targetNode.Name,
 					targetNode.Name,
 				))
+
+				By("Checking that all migrated VMIs have the new pod IP address on VMI status")
+				for _, vmi := range vmis {
+					Eventually(func() error {
+						newvmi, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
+						Expect(err).ToNot(HaveOccurred())
```

Comment:
Please add an expectation string at the end of the expect to make test debugging easier.

Reply:
Added